core-plugins/src/test/java/io/cdap/plugin/batch/joiner/JoinerTestRun.java

```diff
@@ -228,7 +228,7 @@ public void testOuterJoin() throws Exception {
       Schema.Field.of("renamed_category", Schema.nullableOf(Schema.of(Schema.Type.STRING))));
 
     ETLStage joinStage =
-      new ETLStage("joiner",
+      new ETLStage("io/cdap/plugin/file/joiner",
```

Comment:
why are we changing unit tests? isn't this unrelated to e2e tests?

Reply:
yes. I haven't changed it and I guessed it's due to running some maven command and gets changed automatically. will revert these changes.